curator/server.js

```diff
@@ -1,7 +1,6 @@
 const database = require('../database/start.js')
 const app      = require('./app.js')(database)
 const port     = require('../tools/globals.js').curator_port
-
```

Comment:
this is really minor, but we do consistently keep a space between requiring variables and the rest of the code